modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -281,9 +281,9 @@ will only return label `:A`, because that is the only label for which traversal
 [[access-control-limitations-db-operations]]
 == Security and count store operations
 
-The rules of a security model may impact some of the database operations.
-This means extra security checks are necessary to incur additional data accesses, especially in the case of count store operations.
-These are, however, usually very fast lookups and the difference might be noticeable.
+The rules of a security model may impact some database operations.
+This is because extra security checks are necessary to incur additional data accesses, especially in the case of count store operations.
```

Comment:
What do you think about this instead? 
```
This is because extra security checks are necessary and these checks will require additional data accesses, especially in the case of count store operations.

Reply:
I think this is a bit wordy and adds unnecessary complexity. Did you see the two sentences above? I tried to simplify the paragraph.